README.md

```diff
@@ -98,7 +95,21 @@ To preview the exampleSite, run
 ```
 make demo
 ```
- The above command copies current state of the theme to exampleSite/themes and starts hugo with hugo serve -D (Go does not support Symlink directories)
+The above command copies current state of the theme to exampleSite/themes and starts hugo with hugo serve -D (Go does not support Symlink directories)
+
+If you're having trouble running ```make``` commands you may need to install some dependencies for ```build``` to work. If this is the case, try installing the ```npm``` dependencies with the following commands:
+
+```
+npm install -g less
+```
+
+or
+
+```
+npm install -g uglifycss
```

Comment:
You can install both depedencies with one command: `npm install -g less uglifycss`